docs/reference/query-dsl/geo-polygon-query.asciidoc

```diff
@@ -6,25 +6,29 @@ points. Here is an example:
 
 [source,js]
 --------------------------------------------------
+GET /_search
 {
-    "bool" : {
-        "query" : {
-            "match_all" : {}
-        },
-        "filter" : {
-            "geo_polygon" : {
-                "person.location" : {
-                    "points" : [
+    "query": {
+        "bool" : {
+            "must" : {
```

Comment:
Actually I wonder if it'd be better not to use it in the filter context? Like, can you just remove the bool entirely? If it needs the bool we should probably explain why it does. But that can be in another PR I think.